ogre2/src/Ogre2ThermalCamera.cc

```diff
@@ -150,6 +160,10 @@ class ignition::rendering::Ogre2ThermalCameraPrivate
   /// \brief Pointer to material switcher
   public: std::unique_ptr<Ogre2ThermalCameraMaterialSwitcher>
       thermalMaterialSwitcher = nullptr;
+
+  /// \brief Add variation to temperature values based on object rgb values
+  /// This only affects objects that are not heat sources
+  public: bool rgbToTemp = true;
```

Comment:
I was going to add a API for setting this value but realized that could break ign-rendering ABI so I didn't do it. It can remain const for now but we should consider making this configurable in edifice.

Reply:
added todo note 397a443